server/server_test.go

```diff
@@ -507,20 +506,6 @@ func runTestStatusAPI(c *C) {
 	c.Assert(data.GitHash, Equals, printer.TiDBGitHash)
 }
 
-func runTestMultiPacket(c *C) {
```

Comment:
This test fails and doesn't make much sense if we enforce the entry and transaction limit.